src/config.rs

```diff
@@ -42,6 +43,7 @@ impl HoardConfig {
             tertiary_color: Some(Self::default_colors(2)),
             command_color: Some(Self::default_colors(3)),
             parameter_token: Some(Self::default_parameter_token()),
+            read_from_current_directory: Some(false)
```

Comment:
Create the default value for `read_from_current_directory` similarly to the other default values
`read_from_current_directory: Some(Self::default_read_from_current_dir()`

Then, add
`HoardConfig::default_read_from_current_dir()`
Which just returns `false`

Might seem a bit overkill, but I planned to abstract this part out at some point

Reply:
Also, good choice to add the `Option` for the bool, to support older config files :smile: